examples/config.yml

```diff
@@ -1,6 +1,7 @@
 spectaql:
   logoFile: ./test/fixtures/logo.png
   faviconFile: ./test/fixtures/favicon.png
+  scrollPaddingTopPx: 50
```

Comment:
This could also be applied to this padding var here to make the scroll-spy take the padding into account. Could be another pass if difficut

https://github.com/anvilco/spectaql/blob/fc085f851da43b97ff696275bbfe68bca3d81c95/src/themes/default/javascripts/scroll-spy.js#L6